src/content/en/tools/chrome-devtools/inspect-styles/edit-dom.md

```diff
@@ -49,6 +49,42 @@ to go into Inspect Element Mode, then click on an element.
 Use the [`inspect`][inspect] method in the console, such as
 `inspect(document.body)`.
 
+### View the rendered and natural sizes of an image {: #image-sizes }
+
+Hover over an `img` tag in the **DOM Tree** to view the rendered and natural sizes of that
+image.
+
+![Rendered and natural image sizes](imgs/image-sizes.png)
+
+### View which image in a source set (srcset) is being used {: #srcset }
+
+The `srcset` property on `img` elements lets you minimize the amount of bytes that you
```

Comment:
Nit - might be nice to link `srcset` to our srcset docs - https://developers.google.com/web/fundamentals/design-and-ux/responsive/images#images_in_markup

Reply:
Done